app/controllers/application_controller.rb

```diff
@@ -2,4 +2,11 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  # skip_before_filter :verify_authenticity_token
+
+  private
```

Comment:
whitespace: put a new line after private and after the `current_user` method.
Rule of thumb:  `same indent && different concept  == new line`